code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -23,6 +23,7 @@
 	pressure_resistance = 7 * ONE_ATMOSPHERE
 	var/temperature_resistance = 1000 + T0C
 	var/starter_temp
+	var/bluespace = FALSE //is it a canister of holding?
```

Comment:
you should probably override update icon in the subtype instead of using a var like this